### PR TITLE
Add the possibility to upload images to a company.

### PR DESCRIPTION
**Pull request by laurandarius**

### CODE REVIEW

**Comment by laurandarius** [comment]
[Add the possibility to upload images to a company.](https://trello.com/c/G2tLhjmR/8-add-the-possibility-to-upload-images-to-a-company)

File: app/containers/ImageInput/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ImageInputComponent from 'components/ImageInput';
+
+class ImageInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.fileUpload = React.createRef();
+    this.showFileUpload = this.showFileUpload.bind(this);
```
Comment (lgnstn) [inline app/containers/ImageInput/index.js]: Please avoid using `bind` (it creates a new function every time and slows down the app). You can use `(...params) => this.showFileUpload(...params)` instead.

File: app/components/ImageInput/index.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      label={label}
+      setFieldValue={setFieldValue}
+      fullWidth
+    />
+  );
+
+  return (
+    <div>
+      {renderTextField()}
+      <input
```
Comment (lgnstn) [inline app/components/ImageInput/index.js]: Why do you have both a text input and a file input? You need only the file input.

File: app/components/ImageInput/index.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    type,
+    setFieldValue,
+    showFileUpload,
+    handleImageChange,
+    fileUpload,
+  } = props;
+
+  const renderTextField = () => (
+    <TextField
+      InputProps={{
```
Comment (lgnstn) [inline app/components/ImageInput/index.js]: `InputProps` should start with a lower case letter, `inputProps`.

File: app/services/addCompanyRequest.js [hunk 4]
```diff
@@ -2,10 +2,15 @@ import fetch from 'isomorphic-fetch';
 import { API_URL } from 'config';
 
 const addCompanyRequest = async data => {
+  const formData = new FormData();
+  formData.append('name', data.name);
```
Comment (lgnstn) [inline app/services/addCompanyRequest.js]: You can make this code more flexible and open to changes if you iterate through all the keys of the data object and append them to the form.

**Review by lgnstn — CHANGES_REQUESTED** [review]

**Comment by laurandarius** [comment]
@lgnstn updated.

**Review by lgnstn — APPROVED** [review]

**Pull request merged**